ATOMS.md

```diff
@@ -0,0 +1,63 @@
+## Mobiledoc Atoms
+
+Atoms are effectively read-only inline cards.
+
+An atom is an object with a name and at least one of several hooks defined. For example:
+
+```js
+var demoAtom = {
+  name: 'mention',
+  display: {
+    setup(element, options, env, text, payload) {
+      element.innerHTML = `<span class="mention">${text}</span>`;
+    }
+  }
+}
+```
+
+### Available hooks
+
+Like [CARDS](CARDS.md) there are several hooks an atom should define.
+
+```js
+var exampleAtom = {
+  name: 'example',
+  display: {
+    setup(element, options, env, text, payload) {},
+    teardown(setupReturnValue) {}
+  },
+  html: {
```

Comment:
Right, and according to the above these would be two atoms instead of one with two modes:

```js
// package: mda-example
// file: dom.js
export default {
  name: 'example',
  type: 'dom',
  render({element, options, env, text, payload}) {
  }
};
```

```js
// package: mda-example
// file: html.js
export default {
  name: 'example',
  type: 'html',
  render({buffer, options, env, text, payload}) {
  }
};
```